IM_calculation/Advanced_IM/Models/Taghavi_Miranda_2005/run.py

```diff
@@ -0,0 +1,74 @@
+"""
+Python script to run a 3D waveform through Taghavi_Miranda_2005 and store the outputs to a txt file
+"""
+
+import logging
+import os
+
+import numpy as np
+import pandas as pd
+
+from IM_calculation.Advanced_IM import runlibs_2d
+from IM_calculation.IM.Taghavi_Miranda_2005 import Taghavi_Miranda_2005
+from qcore.timeseries import read_ascii
+
+model_dir = os.path.dirname(__file__)
+
+COMPONENT = ["000", "090"]
+ALPHA = [0, 1.5, 5, 15, 30]
+C = [0.02, 0.05, 0.1]
+T = [0.1, 0.4, 0.7, 1.5, 3.0]
+STORIES = 10
+
+
+def main(comp_000, comp_090, output_dir):
+
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
```

Comment:
exist_ok=True?

Reply:
Not actually resolved

Reply:
the newest commit resolved this? o.O?
were you looking at a older commit?